judge/models/submission.py

```diff
@@ -71,7 +71,7 @@ class Submission(models.Model):
     status = models.CharField(verbose_name=_('status'), max_length=2, choices=STATUS, default='QU', db_index=True)
     result = models.CharField(verbose_name=_('result'), max_length=3, choices=SUBMISSION_RESULT,
                               default=None, null=True, blank=True, db_index=True)
-    error = models.TextField(verbose_name=_('compile errors'), null=True, blank=True)
+    error = models.TextField(verbose_name=_('compile errors'), null=True)
```

Comment:
After further discussion it seems it might be best to leave this as-is, since the alternatives all seem worse.